x-pack/platform/plugins/private/reporting/public/redirect/redirect_app.tsx

```diff
@@ -10,15 +10,20 @@ import type { FunctionComponent } from 'react';
 import React, { useEffect, useState } from 'react';
 
 import { EuiCallOut, EuiCodeBlock, UseEuiTheme } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
 import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
 
 import type { ScopedHistory } from '@kbn/core/public';
+import {
+  DASHBOARD_APP_LOCATOR,
+  LENS_APP_LOCATOR,
+  VISUALIZE_APP_LOCATOR,
+} from '@kbn/deeplinks-analytics';
```

Comment:
nit: consider 

- creating and exporting the array from `@kbn/deeplinks-analytics`, e.g. `REPORTABLE_LOCATOR_IDS` or similar, or
- creating a constant array here or elsewhere within reporting.

It will make it easier to find and grok if we enable reporting for some other surface.